cwms-data-api/src/main/java/cwms/cda/api/ForecastInstanceController.java

```diff
@@ -0,0 +1,209 @@
+                            "owning office of the forecast instance to be deleted."),
+                    @OpenApiParam(name = OFFICE, required = true, description = "Specifies the " +
+                            "owning office of the forecast spec associated with the forecast instance" +
+                            "to be deleted."),
+                    @OpenApiParam(name = SPEC_ID, required = true, description = "Specifies the " +
+                            "spec id of the forecast spec associated with the forecast instance" +
+                            "to be deleted."),
+                    @OpenApiParam(name = LOCATION, required = true, description = "Specifies the " +
+                            "location of the forecast spec associated with the forecast instance" +
+                            "to be deleted."),
+            },
+            responses = {
+                    @OpenApiResponse(status = STATUS_404, description = "The provided combination of "
+                            + "parameters did not find a forecast instance."),
+            },
+            path = "/forecast-instance",
+            method = HttpMethod.DELETE,
+            tags = TAG
+    )
+    @Override
+    public void delete(@NotNull Context ctx, @NotNull String forecastSpecId) {
+        try (final Timer.Context ignored = markAndTime(GET_ONE)) {
+            throw new UnsupportedOperationException(NOT_SUPPORTED_YET);
+        }
+    }
+
+    @OpenApi(
+            description = "Used to get all forecast instances for a given forecast spec",
+            queryParams = {
+                    @OpenApiParam(name = OFFICE, required = true, description = "Specifies the " +
+                            "owning office of the forecast spec whose forecast instance is to be " +
+                            "included in the response."),
+                    @OpenApiParam(name = SPEC_ID, required = true, description = "Specifies the " +
+                            "spec id of the forecast spec whose forecast instance data is to be " +
+                            "included in the response."),
+                    @OpenApiParam(name = LOCATION, required = true, description = "Specifies the " +
+                            "location of the forecast spec whose forecast instance data to be included " +
+                            "in the response."),
+            },
+            responses = {
+                    @OpenApiResponse(status = STATUS_200,
+                            description = "A list of elements of the data set you've selected.",
+                            content = {
+                                    @OpenApiContent(from = ForecastInstance.class, type = Formats.JSONV2)}),
+                    @OpenApiResponse(status = STATUS_400, description = "Invalid parameter combination"),
+                    @OpenApiResponse(status = STATUS_404, description = "The provided combination of "
```

Comment:
I believe we generally don't throw 404's on getAll since we're just applying filters. Maybe we should be?

Reply:
i agree it's most likely not needed. done